misk-core/src/main/kotlin/misk/security/ssl/TrustStoreConfig.kt

```diff
@@ -1,8 +1,9 @@
 package misk.security.ssl
 
 import misk.security.ssl.SslLoader.Companion.FORMAT_JCEKS
+import javax.inject.Inject
 
-data class TrustStoreConfig(
+data class TrustStoreConfig @Inject constructor(
```

Comment:
Same Q: are these all provided by factories or providers that invoke the constructor manually?

Reply:
These aren't injected out of the box. Explicit instances are provided by the app from HttpClientsConfig. However, our strict guice mode means that you need to have this annotation, even if you provide it.

Reply:
Ah, that clarifies things. Thanks!

Reply:
Oh wow, that’s a terrible bug. Lets fix